components/List/examples/MDCSelfSizingStereoCellExample.m

```diff
@@ -0,0 +1,163 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCSelfSizingStereoCellExample.h"
+
+#import "MDCSelfSizingStereoCell.h"
+
+static CGFloat const kArbitraryCellHeight = 75.f;
+static NSString *const kSelfSizingStereoCellIdentifier = @"kSelfSizingStereoCellIdentifier";
+static NSString *const kSelfSizingStereoCellExampleComponent = @"List Items";
+static NSString *const kSelfSizingStereoCellExampleDescription =
+    @"Self Sizing Stereo Cell Typical Use";
+
+@interface MDCSelfSizingStereoCellExample () <UICollectionViewDelegate,
+                                      UICollectionViewDataSource>
+
+@property (strong, nonatomic) UICollectionView *collectionView;
+@property (strong, nonatomic) UICollectionViewFlowLayout *collectionViewLayout;
+@property (strong, nonatomic) NSArray *randomStrings;
+@property (nonatomic, assign) NSInteger numberOfCells;
```

Comment:
super nit, but can we place the nonatomic as the 2nd parameter like the above properties?

Reply:
actually noticed that the ordering in the rest of the files is `nonatomic` first, so maybe change the placement in the above properties and leave this one as is.

Reply:
Okie dokie

Reply:
Done in 64652f0f58c0521445e92d50910e32fb9bdee360